index.html

```diff
@@ -639,38 +639,24 @@ <h2>Version <span property="doap:revision">1.0.0</span>, Editor’s Draft, 2022-
-                <p>
-                  A city has public places where I can do all kinds of things,
-                  and also a private house with a private room which may be just
-                  mine. In that house there are spaces where I do things with
-                  family, friends, colleagues. The web must, like a
-                  well-designed building, provide a gradient of intimacy between
-                  the private and the public, so I can easily recognize the
-                  difference, easily know which I am in, and easily welcome
-                  people to come into the more intimate parts, as I want to.
-                </p>
-
-                <footer>
-                  <cite>
-                    <a href="https://www.w3.org/DesignIssues/Gradient" rel="cito:includesQuotationFrom">The Intimacy Gradient</a>
-                    ,
-                    <a href="https://www.w3.org/People/Berners-Lee/card#i"
-                      >timbl</a
-                    >
-                    , <time>2022</time>
-                  </cite>
-                </footer>
-              </blockquote>
-
+              <p>A WebID is a way to identify oneself within various social, conceptual, and contextual relations. Social agents
+                 need to be able to present themselves on the Web and set the circumstances in which their profiles can be used
+                 in a standard manner. A Solid Profile is the representation of a social agent's identity in the Solid ecosystem.
+              </p>
+              <p>This document specifies the discovery and the data model of a Solid Profile. It describes profiles and extension
+                 documents describing an individual's social information and and application-level preferences
+              </p>
+              <p>A Solid profile can include information about the social agent,
+                 such as personal information and links to other resources about or related to the agent. This can consist of
+                 publicly readable data, data restricted to named audiences, or data meant to be accessed only by the social agent
+                 themselves.
+              </p>
               <p>
-                If the web is like a well-designed building, the Solid Profile
-                is its lobby - the place the public can discover which parts of
-                the interior the building's owner has given consent for them to
-                see, where the owner can hide or put on display parts of their
-                identity as they see fit.
+                 This specification aims to provide recommendations and considerations for various circumstances
+                 under which profiles are used or shared, such as the use of multiple profiles, connectivity of profiles, persistence and
+                 evolution of profiles, and anonymous and pseudonymous profiles, among others.
```

Comment:
Perhaps leave off persistence and evolution of profiles for now. We don't have much concrete material on that. Can reintroduce when the time comes.

May want to slip in "preferences" and "social graph" or factors specific to identity.

At that point you can drop "among others". "such as" takes care of it and there is a good list.